rocky/rocky/templates/organizations/organization_detail.html

```diff
@@ -24,9 +24,18 @@ <h1>{% translate 'Organization details' %}</h1>
             </tbody>
           </table>
         </div>
-        <a href="{% url 'indemnification_add' organization.code %}" class="button ghost">{% translate 'Add indemnification' %}</a>
+        <a href="{% url 'indemnification_add' organization.code %}"
+           class="button ghost">{% translate 'Add indemnification' %}</a>
       </div>
     </section>
+    {% if organization.tags.all %}
```

Comment:
I think `organization.tags` would suffice

Reply:
That's probably because even though you check for the condition `organization.tags`, you still have to iterate over `organization.tags.all`?

Reply:
Is {{ organization.tags }} an interator? If so you'd probably need to cast it to a list first.

Reply:
If it's a queryset, then yes querysets are iterable. If it's a model and you're trying to iterate over a relation then it's usually something like `Model.<relation name>_set.all`